src/libs/antares/study/binding_constraint/BindingConstraintTimeSeriesNumbers.h

```diff
@@ -0,0 +1,20 @@
+//
+// Created by marechaljas on 27/04/23.
+//
+
+#pragma once
+
+#include "antares/array/matrix.h"
+#include "yuni/core/system/stdint.h"
+
+namespace Antares::Data {
+
+class BindingConstraintTimeSeriesNumbers {
```

Comment:
This class is very small. Therefore, we can question its utility.

Besides, it would not be necessary with the notion of **BindingConstraintGroup** (see remarks in **src/solver/simulation/timeseries-numbers.cpp**)